recipes/sparsehash-c11/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "sparsehash-c11" %}
+# Using same version as last `sparsehash` version, from which this code is based
+{% set version = "2.0.3" %}
+# Using a commit_ref because there is no version tag
+{% set commit_ref = "5ca6de766db32b3fb08a040636423cd3988d2d4f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://codeload.github.com/sparsehash/{{ name }}/zip/{{ commit_ref }}
+  # Filename is needed so that conda-build can recognize it as a .zip
+  fn: {{ name }}-{{ commit_ref }}.zip
+  sha256: 9f2d7003d30c342acfe95f981bf9dffbf59dc0172dd8a678f9ae4f55411b71c0
+  patches:
+    - 0001-Fix-CMake-compilation-in-Windows.patch  # [win]
+
+build:
+  number: 0
+  script:
+    - mkdir build
+    - cd build
+    - cmake -G "%CMAKE_GENERATOR%" -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ..  # [win]
+    - cmake -DCMAKE_INSTALL_PREFIX="$PREFIX" ..  # [not win]
+    - ctest .
+    - cmake --build . --target install
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -r "${PREFIX}/include/google/sparsehash"  # [not win]
+    - if not exist "%LIBRARY_INC%\google\sparsehash" exit 1  # [win]
+
+about:
+  home: https://github.com/sparsehash/sparsehash-c11
+  license: BSD-3-Clause
```

Comment:
It should be ``BSD-3``

Reply:
It's `BSD-3-Clause`: https://opensource.org/licenses/BSD-3-Clause

![image](https://user-images.githubusercontent.com/955251/58201800-5e4f8680-7cac-11e9-9728-aa069ec91bce.png)